### PR TITLE
26+ compatibility

### PR DESCRIPTION
**Pull request by voloshink**
I'm sure there's a better way to handle the error case, but i'm too dumb

### CODE REVIEW

File: core/src/main/kotlin/gg/destiny/lizard/core/chat/ChatSocket.kt [hunk 1]
```diff
@@ -125,6 +125,9 @@ class ChatSocket(
     val type = Message.of(message.substringBefore(" ", UNKNOWN_TYPE))
     val data = message.substringAfter(' ')
     L { "Message: $message" }
+    if (message.substringBefore(" ") == "ERR") {
```
Comment (xiphirx) [inline core/src/main/kotlin/gg/destiny/lizard/core/chat/ChatSocket.kt]: We already deserialize err messages properly so that we can handle them, so lets remove this.

File: versions.gradle [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ext {
   versions = [
       kotlin     : '1.2.60',
-      agp        : '3.1.0',
+      agp        : '3.2.0',
```
Comment (xiphirx) [inline versions.gradle]: Is this really required for 26 compatibility? If not this should be its own PR.

File: app/src/main/kotlin/gg/destiny/lizard/chat/EmoteSpan.kt [hunk 3]
```diff
@@ -93,7 +92,7 @@ class EmoteSpan(val emote: Emote) : DynamicDrawableSpan(), UpdateAppearance {
       bounds.set(drawable.bounds)
       bounds.offset(x.toInt(), newY.toInt())
       clipBounds.union(bounds)
-      canvas.clipRect(clipBounds, Region.Op.REPLACE)
+      canvas.clipRect(clipBounds)
```
Comment (xiphirx) [inline app/src/main/kotlin/gg/destiny/lizard/chat/EmoteSpan.kt]: Does this produce the same behavior? Can you include a screenshot showing so?

**Comment by voloshink** [comment]
nvm this is too much work, i aint learning how to draw shit lol
